packages/cli/src/file.ts

```diff
@@ -50,9 +50,9 @@ export async function readJSON(file: string) {
   return deps.fs.readJSON(file)
 }
 
-export async function outputJSON(file: string, data: any, options: FS.WriteOptions = {}) {
+export async function outputJSON(file: string, data: any) {
```

Comment:
The options param was causing a type error and, when I investigated, I found that it was not being used, so I'm removing it.